jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -760,7 +780,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *obj_p, /**< the objec
                                                        NULL,
                                                        NULL,
                                                        was_enumerable,
-                                                       was_configurable);
+                                                       is_current_configurable);
```

Comment:
It seems I cannot put a comment on the enclosing if (it's not in the diff'd region). However, `if (is_current_data_descriptor)` is a badly misleading (although functinoally correct) predicate. It should be `if (!is_property_desc_data_descriptor)` or `if (is_property_desc_accessor_descriptor)`, as the `ecma_create_named_*`  calls don't work with "current" (which is already deleted) but with "property".